flink-ml-core/src/main/java/org/apache/flink/ml/common/datastream/DataStreamUtils.java

```diff
@@ -280,6 +280,15 @@ public static <IN, ACC, OUT> DataStream<OUT> aggregate(
      * <p>This method takes samples without replacement. If the number of elements in the stream is
      * smaller than expected number of samples, all elements will be included in the sample.
      *
+     * <p>Technical details about this method: Firstly, the input elements are rebalanced. Then, in
```

Comment:
nit: How about we update the java doc as follows and move the technique details to the implementation part or remove it?

`Performs an approximate uniform sampling over the elements in a bounded data stream. The difference of probablity of two data points been sampled is bounded by O(numSamples * p * p / (M * M)), where p is the paralellism of the input stream, M is the total number of data points that the input stream contains.`

This method takes samples without replacement. If the number of elements in the stream is smaller than expected number of samples, all elements will be included in the sample.

Reply:
Thanks for your comments. I've updated the PR.